### PR TITLE
Document OPENWHISK_TMP_DIR usage

### PR DESCRIPTION
**Pull request by dubee**
## Description

Documents how to prevent configuration errors that occur in the Docker for Mac environment after rebooting the host machine.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [x] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

File: ansible/README.md [hunk 1]
```diff
@@ -60,6 +60,15 @@ An expedient workaround is to add alias for `docker0` network to loopback interf
 sudo ifconfig lo0 alias 172.17.0.1/24
 ```
 
+##### Preserving configuration and log directories on reboot
+By default configuration and log data is stored in `/tmp`. However, Mac cleans the `/tmp` directory on reboot,
+resulting in failures when OpenWhisk tries to start up again. To avoid this problem, export the `OPENWHISK_TMP_DIR`
+variable with a value of `/Users/Shared`.
+
+```
+export OPENWHISK_TMP_DIR="/Users/Shared"
```
Comment (rabbah) [inline ansible/README.md]: it's not just macs.

**Comment by dubee** [comment]
@csantanapr, can you review?

**Comment by dubee** [comment]
@csantanapr, bump.

**Comment by csantanapr** [comment]
Thanks Vadim just saw this.

**Pull request merged**